### PR TITLE
Fix/several small issues

### PR DESCRIPTION
**Pull request by karlbaumhauer**

### CODE REVIEW

File: src/components/cms/bnb-cms-footer/template.ts [hunk 1]
```diff
@@ -23,13 +23,13 @@ export const template = (data: BnbCmsFooterProps): HTMLFragment => {
 						value="Über uns"
 						url="about-us.html"
 					></bnb-link>
+					<bnb-link footer value="Unsere Forderungen" url="claims.html"></bnb-link>
 					<!-- <bnb-link footer value="" url="news.html"></bnb-link> -->
```
Comment (koepferd) [inline src/components/cms/bnb-cms-footer/template.ts]: is this comment still needed?

Comment (karlbaumhauer) [reply src/components/cms/bnb-cms-footer/template.ts]: left it there cause news will come soon...

File: src/scaffolding/faqs.hbs [hunk 2]
```diff
@@ -142,8 +142,8 @@ sollte die Partei für die progressiven Ziele unserer Kandidat*innen offen sein.
     <br>
     Jede*r kann einbringen, was sie*er will. Das kann im Bereich Öffentlichkeitsarbeit, Social Media,
     Organisationsentwicklung, Fundraising, Vernetzung o.ä. sein. Die Koordinator*innen schauen mit Dir gemeinsam, welche
```
Comment (koepferd) [inline src/scaffolding/faqs.hbs]: *dir

File: src/scaffolding/get-started.hbs [hunk 3]
```diff
@@ -39,7 +39,7 @@
 </bnb-cms-text-icon>
 <bnb-cms-text-icon icon="tripleIconset" heading="Nominiere die besten Kandidat*innen!"
     data-resources="[{paths: ['components/cms/bnb-cms-text-icon/index.js']}]"
-    button='{"value":"Schlage jetzt dein*e Kandidat*in vor", "url":"contact.html", "target":"_blank", "version":"primary"}'>
+    button='{"disabled":"true", "value":"Schlage hir ab dem 12.03.2020 dein*e Kandidat*in vor", "url":"contact.html", "target":"_blank", "version":"primary"}'>
```
Comment (koepferd) [inline src/scaffolding/get-started.hbs]: *hier

File: src/scaffolding/home.hbs [hunk 4]
```diff
@@ -57,12 +58,12 @@
         <bnb-bullet-item slot="list" icon="bildmarke"><b>Geschafft!</b></bnb-bullet-item>
     </bnb-bullet-list>
 </bnb-cms-info-box>
-<bnb-cms-youtube heading="lerne uns kennen" youtubeinfo='{"youtubeId":"spYbjeokOBs", "thumbnail":{"src":"../../_assets/img/stage.jpg", "title":"Testbild 16:9", "objectFit":"cover", "objectPosition":"center", "srcset":[{"src":"../../_assets/img/testbild_300.jpeg", "width":300},
+{{!-- <bnb-cms-youtube heading="lerne uns kennen" youtubeinfo='{"youtubeId":"spYbjeokOBs", "thumbnail":{"src":"../../_assets/img/stage.jpg", "title":"Testbild 16:9", "objectFit":"cover", "objectPosition":"center", "srcset":[{"src":"../../_assets/img/testbild_300.jpeg", "width":300},
```
Comment (koepferd) [inline src/scaffolding/home.hbs]: on purpose only commented instead of removed?

Comment (karlbaumhauer) [reply src/scaffolding/home.hbs]: yes
will be there on 12.03.

**Pull request merged**